src/utils.ts

```diff
@@ -0,0 +1,40 @@
+import * as ini from "ini";
+import { promises as fs } from "fs";
+import * as core from "@actions/core";
+import * as path from "path";
+
+async function findStorageDriver(filePaths: string[]): Promise<string> {
+    let storageDriver = "";
+
+    for (const filePath of filePaths) {
+        core.debug(`Checking if the storage file exists at ${filePath}`);
+        if (await fileExists(filePath)) {
+            core.debug(`Storage file exists at ${filePath}`);
+            const fileContent = ini.parse(await fs.readFile(filePath, "utf-8"));
+            storageDriver = fileContent.storage.driver;
```

Comment:
if storage driver is set in the first file but not the second one, then this function will return "", which is not the correct behaviour

Reply:
Yeah, let me correct